### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by wjzijderveld**
Continues on #9 

@ericdowell Is this something you can live with?

### CODE REVIEW

**Comment by ericdowell** [comment]
Looks great.

**Comment by wjzijderveld** [comment]
Cool, @qandidate-labs/toggle who wants to CR/Merge this?

File: lib/Qandidate/Toggle/ToggleManager.php [hunk 1]
```diff
@@ -74,6 +76,55 @@ public function update(Toggle $toggle)
...
 
     /**
+     * Rename the toggle.
+     *
+     * @param string $oldName
+     * @param string $newName
+     *
+     * @throws RuntimeException
+     *
+     * @return bool
```
Comment (othillo) [inline lib/Qandidate/Toggle/ToggleManager.php]: boolean

**Comment by othillo** [comment]
:+1:

**Comment by wjzijderveld** [comment]
Thanks @ericdowell

**Pull request merged**